www/utils/ai.ts

```diff
@@ -1,15 +1,15 @@
+import { getHonchoApp, getHonchoUser, honcho } from '@/utils/honcho';
 import { createClient } from '@/utils/supabase/server';
-import { NextRequest, NextResponse } from 'next/server';
-import { thinkCall, respondCall } from './actions';
-import { honcho, getHonchoApp, getHonchoUser } from '@/utils/honcho';
-import { streamText } from 'ai';
 import { createOpenRouter } from '@openrouter/ai-sdk-provider';
+import { streamText } from 'ai';
+import d from 'dedent-js';
 
 import * as Sentry from '@sentry/nextjs';
 
-export const runtime = 'nodejs';
-export const maxDuration = 100;
-export const dynamic = 'force-dynamic'; // always run dynamically
```

Comment:
Need these lines in the route especially the force dynamic. Otherwise would get weird issues in vercel